server/taskSpreadsheet.go

```diff
@@ -92,7 +92,7 @@ func generateReport(ctx context.Context, aaId int, date time.Time) (err error) {
 	} else if generation, err = checkReportGeneration(ctx, db.Db, aa, forceGeneration); err != nil || !generation {
 	} else if updateId, err = registerAccountReportGeneration(db.Db, aa); err != nil {
 	} else {
-		errs := reports.GenerateReport(ctx, aa, date)
+		errs := reports.GenerateReport(ctx, aa, nil, date)
```

Comment:
Why do you pass `nil` here ?

Reply:
Third argument is a list of sub-accounts. It is used for master spreadsheets (cf. `taskMasterSpreadsheet.go`).